internal/manifests/opampbridge/labels.go

```diff
@@ -0,0 +1,74 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opampbridge
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/open-telemetry/opentelemetry-operator/apis/v1alpha1"
+	"github.com/open-telemetry/opentelemetry-operator/internal/naming"
+)
+
+func isFilteredLabel(label string, filterLabels []string) bool {
+	for _, pattern := range filterLabels {
+		match, _ := regexp.MatchString(pattern, label)
+		return match
+	}
+
+	return false
+}
+
+// Labels return the common labels to all objects that are part of a managed OpAMPBridge.
+func Labels(instance v1alpha1.OpAMPBridge, name string, filterLabels []string) map[string]string {
```

Comment:
This should be updated to https://github.com/open-telemetry/opentelemetry-operator/pull/2238

Reply:
It would be also possible to reuse the function by supplying objectmeta as the first parameter.

Reply:
> It would be also possible to reuse the function by supplying objectmeta as the first parameter.

https://github.com/open-telemetry/opentelemetry-operator/pull/2254

Reply:
> This should be updated to #2238

Done.